.pre-commit-config.yaml

```diff
@@ -22,7 +22,7 @@ repos:
     hooks:
       - id: validate-pyproject
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.6.8
+    rev: v0.11.5
```

Comment:
The `ruff` in `pre-commit` and the `ruff` in the `pyproject.toml` were different versions and were formatting files differently.